native/core/src/execution/operators/scan.rs

```diff
@@ -365,28 +378,23 @@ struct ScanStream<'a> {
     scan: ScanExec,
     /// Schema representing the data
     schema: SchemaRef,
-    /// Metrics
```

Comment:
is it dropped because it repeats what we have on SparkPlan?

Reply:
I have reverted some of these changes now